packages/svelte-docgen/src/analyzer/prop.js

```diff
@@ -3,23 +3,33 @@
  */
 
 import path from "pathe";
+import { isTypeRef } from "../doc/utils.js";
+
+/** @typedef { Omit<Doc.Prop, 'type'> & { type: Doc.Type } } Prop */
```

Comment:
Got a little lost there; why are we excluding `TypeRef` here?

Reply:
I don’t remember writing this. Could it have been generated by VS Code’s refactoring tool????

Reply:
Removed!